.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this something that we should ignore? I feel like this is a cache of previous test failures. The standard Python gitignore ignores this directory: https://github.com/github/gitignore/blob/master/Python.gitignore#L48